From 2c7bbd7ce02be70a2af21e45a17c2cdca187526f Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Mon, 20 May 2019 18:55:18 -0600
Subject: [PATCH 4/4] Support application/gzip MIME type in extraction

file 5.37 changed the gzip MIME type from application/x-gzip to
application/gzip, so support this when checking to extract source files.
---
 scripts/libmakepkg/source/file.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/libmakepkg/source/file.sh.in b/scripts/libmakepkg/source/file.sh.in
index 08b8bef8..8492ba11 100644
--- a/scripts/libmakepkg/source/file.sh.in
+++ b/scripts/libmakepkg/source/file.sh.in
@@ -102,7 +102,7 @@ extract_file() {
 	case "$file_type" in
 		*application/x-tar*|*application/zip*|*application/x-zip*|*application/x-cpio*)
 			cmd="bsdtar" ;;
-		*application/x-gzip*)
+		*application/x-gzip*|*application/gzip*)
 			case "$ext" in
 				gz|z|Z) cmd="gzip" ;;
 				*) return;;
-- 
2.21.0

